### ISSUE TITLE
 Fine-turn the above Softmax model with Triplet loss.

### ISSUE THREAD

**Issue by zhaowwenzhong** [deepinsight/insightface#227]
在train_softmax.py中好像没有tripletloss， 在train.py中有tripletloss。但你的例子中写的是tripletloss
(4). Fine-turn the above Softmax model with Triplet loss.
CUDA_VISIBLE_DEVICES='0,1,2,3' python -u train_softmax.py --network m1 --loss-type 12 --lr 0.005 --mom 0.0 --per-batch-size 150 --data-dir ../datasets/faces_ms1m_112x112 --pretrained ../model-m1-softmax,50 --prefix ../model-m1-triplet
train.py是不是可以使用？

**Comment by nttstar** [comment]
Yes use train.py.

**Comment by Edwardmark** [comment]
@nttstat 大侠请问您试过用triplet loss微调吗？请问为啥这里动量设为0而不是0.9？还有请问如何能让triplet loss收敛呢？我训练了很久，一直不收敛。
